### PR TITLE
Initial commit for e2e tests using cypress for Feedback tests

### PR DESCRIPTION
**Pull request by supriyagawade**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [ ] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [ ] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by gautamkrishnar** [comment]
CodeQL scanning must be removed for PRs, Opened a PR to fix this failing test at https://github.com/1-Platform/one-platform/pull/862

File: package.json [hunk 1]
```diff
@@ -14,21 +14,10 @@
     "url": "git+https://github.com/1-Platform/one-platform.git"
   },
   "version": "0.0.0",
-  "author": {
-    "name": "Sayak Sarkar",
-    "email": "sayak.bugsmith@gmail.com"
-  },
+  "author": "Sayak Sarkar <sayak.bugsmith@gmail.com>",
```
Comment (gautamkrishnar) [inline package.json:17]: @supriyagawade could you please explain how this change is related to cypress implementation?

File: e2e/integration/feedbacktests.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        cy.visit( 'http://stage.one.redhat.com/' );
+        Cypress.config( 'includeShadowDom', true );
+        cy.get( '#username', { timeout: 5000 } ).type( 'one-portal-test' );
+        cy.get( '#password' ).type( 'iequeivaequ:e2OobahM3vahbaePh8' );
+        cy.get( '#submit' ).click();
+        //cy.contains( 'Feedback', { timeout: 5000 } ).click();
+
+    } );
+    Cypress.on( 'uncaught:exception', ( err, runnable ) => {
+        // returning false here prevents Cypress from
```
Comment (gautamkrishnar) [inline e2e/integration/feedbacktests.js:15]: Why this is required? These are supposed to fail right when something is changed?

**Comment by gautamkrishnar** [comment]
Also please add  a script under package.json named `e2e` so that the test can be run via npm/yarn:
```json
"e2e":"cypress run"
```

**Comment by riginoommen** [comment]
Feedback SPA has changed and the SPA has migrated to vue

**Pull request closed without merging**